File: Typescript/test/findPermutationDifference.test.ts
import { expect, test } from '@jest/globals';
import * as solution from '../src/findPermutationDifference';

test('findPermutationDifference Example 1', () => {
	let input0 = "abc";
	let input1 = "bac"
	let expected = 2;
	expect(solution.findPermutationDifference(input0, input1)).toBe(expected);
});

test('findPermutationDifference Example 2', () => {
	let input0 = "abcde";
	let input1 = "edbac"
	let expected = 12;
	expect(solution.findPermutationDifference(input0, input1)).toBe(expected);
});

File: Dart/test/findPermutationDifference_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:leetcode/findPermutationDifference.dart';

void main() {
  test("test example 1", () {
    var input0 = "abc";
    var input1 = "bac";
    var expected = 2;

    final sln = Solution();
    var output = sln.findPermutationDifference(input0, input1);
    expect(output, expected);
  });
  test("test example 2", () {
    var input0 = "abcde";
    var input1 = "edbac";
    var expected = 12;

    final sln = Solution();
    var output = sln.findPermutationDifference(input0, input1);
    expect(output, expected);
  });
}
